app/package.json

```diff
@@ -26,6 +26,7 @@
     "@sentry/browser": "5.22.3",
     "@types/lodash": "^4.14.168",
     "bip39-light": "^1.0.7",
+    "bitbox02-api": "^0.10.0",
```

Comment:
@benma it seems the library is still being loaded within the main js bundle - did you face any issues if/when you tried it?

Reply:
@thisconnect @x1ddos and I were trying to accomplish this, but so far our attempts have not been successful.
Our playground with some attempts is here: https://github.com/thisconnect/adalite/commits/notsure - it seems like the bitbox02 part is moved to a different file, but we can't figure out how to also remove it from the main bundle. Maybe you have an idea?

It also seems the output of GopherJS (which is how most of bitbox02-api is generated) does not play well with webpack and code splitting. See also this discussion: https://github.com/gopherjs/gopherjs/issues/524#issuecomment-956265776

At this point we are not sure if it is even possible in the current configuration, but we are still trying. It should become much easier once we rewrite the bitbox02-api package without GopherJS (by that point the package size will also be much smaller anyway).

We would appreciate it a lot if, in the worst case, you could merge and deploy this PR even without code splitting for the time being, and we provide a solution later. It would be unfortunate if this held up the release for too long, as our customers are eagerly awaiting Cardano support. What do you think?

Reply:
interesting, will look into that, meanwhile I found an ancient commit where we successfully implemented code splitting (though abandoned it later on), you can check if you find anything useful there: https://github.com/vacuumlabs/adalite/commit/6a4c8e9ce64e589668d537b42c83add3e7dfe8ec#

Reply:
I managed to make code splitting work in a straight forward way, but it will require to upgrade to webpack 5. I did this locally and fortunately, the change is minimal and everything seems to work.

I will prepare a few commits soon and document them extensively.

Reply:
I just added two commits, `upgrade from webpack 4 to webpack 5` and `bitbox02: lazily load bitbox02-api and split code out of main bundle`.

Please check it out @refi93. The dist/js folder now has two js bundles, main and bitbox02-api. The latter is only loaded when unlocking a BitBox02, as can be verified by checking the network tab in the developer tools.